src/test/munit/reconnection/send-and-receive-reconnection-test-case.xml

```diff
@@ -33,7 +33,7 @@
                         <munit-tools:value ><![CDATA[#[1]]]></munit-tools:value>
             </munit-tools:queue>
 
-            <scripting:execute engine="nashorn" >
+            <scripting:execute engine="groovy" >
```

Comment:
Why did the engine change?

Reply:
@tomastoloza nashorn is no longer packaged with Java 17 jdk. So he would need to add an external library to make the test work and there are no existing tests for that on the scripting module yet, that would also require to use the external nashorn library only for java 17 as it's hard to find a dependency in maven central which supports older versions and doesn't require the special -add-modules vm parameter (I'm working on that). Apparently @marianborzik was able to make the same test code work with Groovy and for the purpose of the testing this should be fine.